packages/features/bookings/Booker/components/Header.tsx

```diff
@@ -179,17 +179,32 @@ const LayoutToggle = ({
     return [
       {
         value: BookerLayouts.MONTH_VIEW,
-        label: <Icon name="calendar" width="16" height="16" />,
+        label: (
+          <>
+            <Icon name="calendar" width="16" height="16" />
+            <span className="sr-only">${t("switch_monthly")}</span>
+          </>
```

Comment:
Hey @mfranzke Instead, can we set `aria-label` to the icon?

Reply:
@anikdhabal good idea, but it doesn't work as the icons get an `aria-hidden="true"`-attribute added to their `svg`-HTML-elements so that this `aria-label`-attribute wouldn't have any affect.

Reply:
Hmm, I just noticed. What if we add an aria label to the button with a common name like 'Toggle Item'?

Reply:
> Hmm, I just noticed. What if we add an aria label to the button with a common name like 'Toggle Item'?

Actually adding an `aria-label` to the button directly would be the best from my point of view, but it's important to still keep the differentiable part that we show as tooltips (like in the screenshot below), as especially screenreader users elsewhere wouldn't know that this toggle is all about (especially if all three buttons would be including "Toggle Item" as a phrase).

<img width="281" alt="image" src="https://github.com/user-attachments/assets/7422e694-bc97-4baa-8a9e-2213ee65d7ec">